learn/react-native/device-biometric-authentication.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Device Biometric Authentication"
+id: "device-biometric-authentication"
+sidebar_label: "Device Biometric Authentication"
```

Comment:
sidebar_label: "Biometric Authentication"

Reply:
Can we have different title and side bar label?